asset-transfer-basic/chaincode-java/.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
+.gradle/
```

Comment:
WARNING:  some of the comments may be more thinking out loud, so discussion points rather than fully formed ideas.

Reply:
Don't ignore good `.ideas` 
👍